src/index.html

```diff
@@ -36,12 +61,23 @@
     <!-- JavaScript and Styles are inserted here -->
   </head>
   <body>
-    <main id="app" class="slds-grid slds-grid_vertical"{% if user.is_authenticated %} data-user="{{ user|serialize }}"{% endif %}>
+    <div
```

Comment:
This is the important change in this file, the rest is whitespace from the linter.